protocol/scripts/deploys/mainnet/v2/05-otc-offer/99-post-deploy.ts

```diff
@@ -0,0 +1,79 @@
+import "@nomiclabs/hardhat-ethers";
+import { ethers } from "hardhat";
+import {
+  ensureExpectedEnvvars,
+  mine,
+} from "../../../helpers";
+import { connectToContracts, getDeployedContracts } from "../contract-addresses";
+import { getDeployedContracts as getOtcDeployedContracts } from "../../otcOffer/contract-addresses";
+
+
+async function main() {
+    ensureExpectedEnvvars();
+    const [owner] = await ethers.getSigners();
+    const deployedContracts = getDeployedContracts();
+    // const multiOtcOffer = MultiOtcOffer__factory.connect(deployedContracts.CORE.)
+    const connectedContracts = connectToContracts(owner);
+    const multiOtcOffer = connectedContracts.STRATEGIES.FOHMO_GNOSIS_STRATEGY.OTC_OFFER.MULTI_OTC_OFFER;
+    const daiToken = deployedContracts.EXTERNAL.MAKER_DAO.DAI_TOKEN;
+    const ohmToken = deployedContracts.EXTERNAL.OLYMPUS.OHM_TOKEN;
+    const gOhmToken = deployedContracts.EXTERNAL.OLYMPUS.GOHM_TOKEN;
+    const otcDeployedContracts = getOtcDeployedContracts();
+    const fundsOwner = otcDeployedContracts.OTC_OFFER.FUNDS_OWNER;
+
+    /// Add OTC markets
+    let otcMarketInfo;
+    {   
+      // dai-ohm. `userBuyToken` is DAI
+      otcMarketInfo = {
+          fundsOwner: fundsOwner,
+          userBuyToken: daiToken,
+          userSellToken: ohmToken,
+          offerPricingToken: 0, // userBuyToken
+          minValidOfferPrice: ethers.utils.parseEther("11"), // minOfferPrice
+          maxValidOfferPrice: ethers.utils.parseEther("12"), //maxOfferPrice
+          scalar: 0, // default scalar
+          offerPrice: ethers.utils.parseEther("11.33") // offerPrice
```

Comment:
I think these need to be updated to be the latest values that we have onchain.

Eg for this, I think it's 11.8
https://etherscan.io/address/0x687A4B0Ac18Ed3796D55E6A1d747bD75591a8bac#readContract

Can you double check the rest too please?